code/modules/projectiles/gun.dm

```diff
@@ -218,7 +219,7 @@
 		user.update_inv_l_hand(0)
 	else
 		user.update_inv_r_hand(0)
-
+	feedback_add_details("gun_fired","[src.name]")
```

Comment:
Recommend you replace with type or some other identifier instead of name, name is often changed by admins or code